src/apps/filters/h3ToLocalIj.c

```diff
@@ -47,32 +47,56 @@ void doCell(H3Index h, H3Index origin) {
     }
 }
 
-int main(int argc, char *argv[]) {
-    // check command line args
-    if (argc != 2) {
-        fprintf(stderr, "usage: %s [origin]\n", argv[0]);
-        exit(1);
-    }
+int main(int argc, char* argv[]) {
+    H3Index origin = 0;
+    H3Index index = 0;
 
-    H3Index origin;
+    Arg helpArg = {.names = {"-h", "--help"},
```

Comment:
This is a total nit, but declaring each arg as a variable like this leads to inconsistent indentation, which makes the arg list far less legible.

I think this might read better as:
```
Arg* args = [
  {.names = {"-h", "--help"},
  ...
];
// Then name the args
Arg helpArg = args[0]; 
```
This avoids naming args you don't need to reference as vars, and keeps the indentation consistent.

Reply:
I am not that convinced, I think that indexing into the args array could lead to errors (off by one, miscounting the number of arguments, etc).